README.md

```diff
@@ -123,10 +123,11 @@ Fetches information about a bot. Does not require authorization.
 |website|`str`|Link to the bot's website|
 |github|`str`|Link to the bot's GitHub repository|
 |short_description|`str`|Short description of the bot|
+|long_description|`str`|Long description of the bot|
 |prefix|`str`|The bot's prefix|
 |invite_url|`str`|Link to the invite URL of the bot|
-|support_server|`str`|Bot support guild invite code|
-|tags|List[`str`]|List of bot tags on the Blist website|
+|support_server|`str`|Bot's support guild code slug|
```

Comment:
This sounds straight-up confusing. I would rephrase it to something else. If you cannot come up with a phrase that sounds better, I'm fine merging as-is.

Reply:
`Bot's support server invite code`

Reply:
server's* ?

Reply:
`guild` is used instead of `server` to remove any ambiguity associated with a Discord guild and an actual server.

Reply:
```suggestion
|support_server|`str`|The bot's support guild invite code|
```
How about this?

Reply:
Looks good to me.